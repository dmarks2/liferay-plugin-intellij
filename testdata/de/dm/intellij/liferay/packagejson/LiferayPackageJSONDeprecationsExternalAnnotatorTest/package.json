{
  "name": "my-module",
  "devDependencies": {
    "liferay-frontend-css-common": <warning descr="Mismatched liferay-frontend-css-common Dependency version (1.0.0 does not match Liferay 7.4.3.55, should be 9.9.9)">"1.0.0"</warning>,
    "liferay-theme-tasks": <warning descr="Mismatched liferay-theme-tasks Dependency version (1.0.0 does not match Liferay 7.4.3.55, should be 9.9.9)">"1.0.0"</warning>
  }
}
