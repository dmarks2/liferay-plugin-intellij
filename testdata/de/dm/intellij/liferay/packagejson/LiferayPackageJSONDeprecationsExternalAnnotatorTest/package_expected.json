{
  "name": "my-module",
  "devDependencies": {
    "liferay-frontend-css-common": "9.9.9",
    "liferay-theme-tasks": "9.9.9"
  }
}
